packages/editor-ui/src/components/mixins/pushConnection.ts

```diff
@@ -381,7 +381,7 @@ export const pushConnection = mixins(
 							return this.restApi().getNodesInformation(nodesToBeFetched);
 						})
 						.then((nodesInfo) => {
-							this.$store.commit('updateNodeTypes', nodesInfo);
+							this.$store.commit('nodeTypes/updateNodeTypes', nodesInfo);
 						});
```

Comment:
maybe this whole fetch and commit needs to be moved into a dispatch in the store module since it seems related
also maybe we can come up with a better name `getFullNodesProperties` for example or something like that